spec/models/manageiq/providers/ansible_tower/automation_manager/configuration_script_spec.rb

```diff
@@ -13,9 +13,13 @@
   end
 
   context "ansible configuration_script" do
-    let(:provider_with_authentication)       { FactoryBot.create(:provider_ansible_tower, :with_authentication) }
-    let(:manager_with_authentication)        { provider_with_authentication.managers.first }
-    let(:manager_with_configuration_scripts) { FactoryBot.create(:automation_manager_ansible_tower, :provider, :configuration_script) }
+    let(:provider_with_authentication) { FactoryBot.create(:provider_ansible_tower, :with_authentication) }
+    let(:manager_with_authentication)  { provider_with_authentication.managers.first }
```

Comment:
> `provider_with_authentication.managers.first`

This could be `provider_with_authentication.automation_manager`

Reply:
@agrare Yes, that's nicer, updated.